doc/manual/packages.rst

```diff
@@ -570,11 +570,36 @@ going to have to do a *force push*:
   indicates that this should replace the ``fixbar`` branch found at
   ``myfork``.
 
-
 Creating a new Package
 ----------------------
 
-Guidelines for Naming a Package
+REQUIRE speaks for itself
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should have a ``REQUIRE`` file in your package repo, with a bare minimum directive of what Julia
+version you expect your users to be running for the package to work. When Julia version ``0.x`` is released,
+use ``julia 0.x`` in your ``REQUIRE`` file. Typically, once a stable release is officially released,
+developers abaondon support for previous versions in future releases of their packages.
```

Comment:
Maybe a more delicate way of saying this would be "Package developers should set, as a floor, the lowest version of Julia they regularly test their package against. Travis-CI (link) enables automatic testing against multiple versions of Julia, and `Pkg.generate` pre-populates a configuration file for it."